Engine/Module/ModuleResourceManager.cpp

```diff
@@ -95,6 +95,7 @@ std::pair<bool, std::string> ModuleResourceManager::InternalImport(const File& f
 	}
 	if (file.file_type == FileType::TEXTURE)
 	{
+		std::lock_guard<std::mutex> lock(thread_comunication.thread_mutex);
```

Comment:
What is being changed exactly in this commit?

Reply:
DevIL is not thread safe and crash if you try to import multiple things at the same time. That is why I added a mutex